aim/tests/unit/test_aim_manager.py

```diff
@@ -2773,3 +2774,57 @@ class TestServiceRedirectHealthGroup(TestServiceRedirectHealthGroupMixin,
                                      TestAciResourceOpsBase,
                                      base.TestAimDBBase):
     pass
+
+
+class TestQosRequirementMixin(object):
```

Comment:
We need to add the test cases in aim/tests/unit/tools/cli/test_manager.py also.

Reply:
Done

Reply:
Just curious, is there a reason why TestManagerResourceOpsBase is not included in your test_manager.py test class?